concordium-node/src/utils.rs

```diff
@@ -296,194 +278,6 @@ pub fn generate_bootstrap_dns(
-                                                                )
-                                                            })
-                                                            .collect::<Vec<_>>();
-                                                        match serialize_bootstrap_peers(
-                                                            &content_peers,
-                                                        ) {
-                                                            Ok(content) => {
-                                                                if public_key
-                                                                    .verify(
-                                                                        content.as_bytes(),
-                                                                        &signature,
-                                                                    )
-                                                                    .is_ok()
-                                                                {
-                                                                    Ok(ret)
-                                                                } else {
-                                                                    Err("Signature invalid")
-                                                                }
-                                                            }
-                                                            Err(_) => {
-                                                                Err("Couldn't reverse encode \
-                                                                     content")
-                                                            }
-                                                        }
-                                                    } else {
-                                                        Err("Not correct length for signature")
-                                                    }
-                                                }
-                                                Err(_) => Err("Could not read signature"),
-                                            }
-                                        }
-                                        Err(_) => Err("Incorrect number of nodes"),
-                                    }
-                                }
-                                Err(_) => Err("Incorrect public key"),
-                            }
-                        } else {
-                            Err("Incorrect size of data")
-                        }
-                    }
-                    Err(_) => Err("Invalid data"),
-                }
-            }
-            Err(_) => Err("Invalid data"),
-        }
-    } else {
-        Err("Missing bytes")
-    }
-}
-
 pub fn generate_ed25519_key() -> SecretKey { SecretKey::generate(&mut OsRng::default()) }
```

Comment:
This could also be moved to `concordium-node/src/bin/genkey.rs` as it is the only place being used.

Reply:
I do not wish to move around things that are not related to the PR.